acceptance/bundle/unitycatalog/capture_schema_dependency/dlt_schema/script

```diff
@@ -0,0 +1 @@
+$CLI bundle validate -o json | jq .resources.pipelines | jq 'map_values(del(.deployment, .permissions))' > out.json
```

Comment:
Could you add a small comment there what do you expect to see in the output.

Also it should be clear -- is the output correct or is there something we want to fix?

Reply:
There's nothing to fix. I'd assume that the tests here are for correct behaviour by default. Should we explicitly call that out in all acceptance tests?

Reply:
I see. Normally, it's not needed, but ${resources.*} behaviour is a bit surprising, so worth calling out.

Reply:
Added a comment for the expectation with the test itself.